### ISSUE TITLE
Rule proposal: prefer-modern-dom-apis - .replaceChildren()

### ISSUE THREAD

**Issue by clshortfuse** [sindresorhus/eslint-plugin-unicorn#2053]
### Description

Instead of iterating all children in a DOM Node, we can use a zero-argument `replaceChildren()` call.

### Fail

```js
while (node.lastChild) {
  node.lastChild.remove();
}
```

```js
while (node.firstChild) {
  node.firstChild.remove();
}
```

### Pass

```js
node.replaceChildren();
```

```js
node.replaceChildren();
```

### Additional Info

Not the same if iterating `node.firstChildElement.remove()` or iterating `node.lastChildElement.remove()` because the DOM structure can look like this:

````html
<div>
  Text Block 1
  <input />
  Text Block 2
</div>
````

**Comment by sindresorhus** [comment]
Accepted
